com.google.Chrome.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: com.google.Chrome
+desktop-file-name-prefix: '(Testing) '
```

Comment:
Is this needed? we can otherwise just push to beta instead of stable directly

Reply:
Is the prefix automatically added for beta apps? (The intent was to push to beta regardless, which I mentioned in the main PR message.)